### PR TITLE
Extend KDP docs with more information on creating a new service

### PR DESCRIPTION
**Pull request by embik**
Adds some screenshots, more details on what to provide for a new service and the API way to create a new service. I'll follow up with more information on the servlet installation, but this is step by step for now.

### CODE REVIEW

File: content/developer-platform/tutorials/your-first-service/_index.en.md [hunk 1]
```diff
@@ -37,9 +83,16 @@ the Service.
...
-(TODO: Make this kubeconfig available via the KDP Dashboard.) You can also use `kubectl` to navigate
-into your workspace and inspect your Service object. In `spec.kubeconfig` you will find the name of
-the kubeconfig Secret that you can use for your Servlet.
+The generated kubeconfig is available from the service's context menu in the Dashboard.
+
+![Service Context Menu](/img/developer-platform/common/service-context-menu.png)
+
+Click "kubeconfig" to inspect and download the kubeconfig:
+
+![Kubeconfig View](/img/developer-platform/common/kubeconfig-view.png)
```
Comment (xrstf) [inline content/developer-platform/tutorials/your-first-service/_index.en.md:92]: Please do not put images in static/img/, we _just_ undid all this stuff and got rid of the "common" (but not actually common) images.

Comment (xrstf) [reply content/developer-platform/tutorials/your-first-service/_index.en.md:92]: cf. https://github.com/kubermatic/docs/pull/1654

Comment (embik) [reply content/developer-platform/tutorials/your-first-service/_index.en.md:92]: Forgot about that.

File: content/developer-platform/tutorials/your-first-service/_index.en.md [hunk 2]
```diff
@@ -25,9 +25,55 @@ for more information from the standpoint of Service Consumers (i.e. endusers).
...
+The service creation wizard also offers several optional metadata fields that will help service consumers
+understand what kind of services are offered. You can (optionally) provide a **documentation link** which
+will be shown to consumers as a way to discover more information about the service, a **logo** which will
+be displayed alongside the service in the service catalog and a markdown-capable **description** to convey
+what the service is about.
+
+### Via API
+
+It is of course also possible to create services from the kcp API directly via `kubectl` or similar tools
+(useful when the service should e.g. managed via GitOps). Here is an example YAML specification:
```
Comment (xrstf) [inline content/developer-platform/tutorials/your-first-service/_index.en.md:47]: `should _be_` or `is`

**Review by xrstf — APPROVED** [review]
/approve

**Pull request merged**